lib/pactas_itero/response/raise_error.rb

```diff
@@ -7,13 +7,13 @@ module Response
 
     # This class raises an PactasItero-flavored exception based
     # HTTP status codes returned by the API
-    class RaiseError < Faraday::Response::Middleware
+    class RaiseError < Faraday::Middleware
 
-      private
-
-      def on_complete(response)
-        if error = PactasItero::Error.from_response(response)
-          raise error
+      def call(request_env)
```

Comment:
Die Leerzeile kann noch weg.

In dem verlinkten Guide steht unten: 

> Do I need to override #call?
For the majority of middleware, it’s not necessary to override the #call method. You can instead use #on_request and #on_complete.

weswegen auch `#on_complete` nach wie vor praktikabel wäre. Bei der Variante mit `#call` steht noch:

> When you do so, remember to call either app.call(env) or super to avoid breaking the middleware stack call!

Da du das tust, ist diese Variante aber so auch ok.